queries/gularen/injections.scm

```diff
@@ -0,0 +1,28 @@
+(code_block_labeled
+  (code_block_label) @injection.language
+  (code_block_content) @injection.content)
+
+; common aliases
+(code_block_labeled
+  (code_block_label) @_language
+  (#any-of? @_language "js")
+  (code_block_content) @injection.content
+  (#set! injection.language "javascript"))
```

Comment:
I don't think we want this. The correct way to do this is for users to use `vim.treesitter.language.register()` to register `js` as an alias for `javascript` etc; then you can just use `(code_block_label)` as `@injection.language` directly.